packages/message-broker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@user-office-software/duo-message-broker",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
Just a small addition here: You should run `npm run lerna:prePublish` after you update package version so the package-lock.json gets updated as well.

Reply:
I did run the script and the package-lock.json was updated with the correct version number, you can see in the diff. If I miss something please let me know :)

Reply:
Sorry, my bad I missed it. Approved!